{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/templates/components/button.jinja

```diff
@@ -25,19 +25,32 @@
     {% else %}
         {% set class = "{} text-{}-content bg-{} hover:bg-{}-focus".format(class, color, color, color) %}
     {% endif %}
-    <button type="{{ type }}"
+    {% if link == "" %}
+        <button type="{{ type }}"
             x-on:click="$dispatch('{{ dispatch_name }}', '{{ variant }} {{ color }} {{ shape }} {{ size }}')"
-            class="{{ class }}">
-        {{ button_content }}
-    </button>
+            {% if disabled == "true" %}
+            disabled
+            {% endif %}
+            class="{{ class }} {{ extra_class }}">
+            {{ button_content }}
+        </button>
+    {% else %}
+        <a
+            href="{{ link }}"
+            {% if target != "" %}target="{{ target }}"{% endif %}
+            {% if target != "" %}rel="{{ rel }}"{% endif %}
```

Comment:
is this suppsoed to be if rel != ""

Reply:
No because I was thinking the rel is only really necessary if the target is included since that's the only time I ever use it.